Assets/MRTK/Core/Inspectors/MRTK.Inspectors.asmdef

```diff
@@ -6,7 +6,10 @@
         "Microsoft.MixedReality.Toolkit.Editor.BuildAndDeploy",
         "Microsoft.MixedReality.Toolkit.Editor.ClassExtensions",
         "Microsoft.MixedReality.Toolkit.Editor.Utilities",
-        "Unity.TextMeshPro.Editor"
+        "Unity.TextMeshPro.Editor",
+        "Unity.TextMeshPro",
+        "Unity.XR.Management",
+        "Unity.XR.Management.Editor"
```

Comment:
I'm not sure if I see XR Management or `MSFT_OPENXR` used in this assembly. Can these be removed or did I miss something?